src/orchard/complete.clj

```diff
@@ -0,0 +1,25 @@
+(ns orchard.complete
+  (:require [cljs-tooling.complete :as cljs-complete]
```

Comment:
This entire library has to be integrated into `orchard`, as it was developed for a similar purpose and it got just to two features - completion and var-info.